charts/kubernetes/templates/_helpers.tpl

```diff
@@ -141,8 +141,8 @@ Metrics
 - name: memory
   lookup:
     prometheus:
-    - query: 'sum(container_memory_working_set_bytes{container!=""{{.Values.prometheusLabels}}}) by (pod)'
-      url: {{ .Values.prometheusURL | quote }}
+    - query: 'sum(container_memory_working_set_bytes{container!=""{{.Values.prometheus.labels | default .Values.prometheusLabels}}}) by (pod)'
```

Comment:
I think these queries also need to move to values - and then we `tpl` it in here.